### ISSUE TITLE
repeated edges in dependency graph

### ISSUE THREAD

**Issue by marcovzla** [clulab/processors#355]
I am trying to parse the sentence `It can detect properties in Java, C, C++, Perl, and IOA programs, in spreadsheet files, and in other data sources.` and I noticed some repeated edges in the dependency graph. Notice the outgoing edges from token 2 to token 0.

```
roots: 2
outgoing:
	0:
	1:
	2: (0,nsubj) (0,nsubj) (0,nsubj) (0,nsubj) (0,nsubj) (1,aux) (2,conj_and) (2,conj_and) (2,conj_and) (2,conj_and) (3,dobj) (5,nmod_in) (13,cc) (19,nmod_in) (21,cc) (25,nmod_in) (26,punct)
	3:
	4:
	5: (4,case) (6,punct) (7,conj) (8,punct) (9,conj) (10,punct) (11,conj) (12,punct) (15,conj) (16,punct) (20,punct)
	6:
	7:
	8:
	9:
	10:
	11:
	12:
	13:
	14:
	15: (14,compound)
	16:
	17:
	18:
	19: (17,case) (18,compound)
	20:
	21:
	22:
	23:
	24:
	25: (22,case) (23,amod) (24,compound)
	26:
incoming:
	0: (2,nsubj) (2,nsubj) (2,nsubj) (2,nsubj) (2,nsubj)
	1: (2,aux)
	2: (2,conj_and) (2,conj_and) (2,conj_and) (2,conj_and)
	3: (2,dobj)
	4: (5,case)
	5: (2,nmod_in)
	6: (5,punct)
	7: (5,conj)
	8: (5,punct)
	9: (5,conj)
	10: (5,punct)
	11: (5,conj)
	12: (5,punct)
	13: (2,cc)
	14: (15,compound)
	15: (5,conj)
	16: (5,punct)
	17: (19,case)
	18: (19,compound)
	19: (2,nmod_in)
	20: (5,punct)
	21: (2,cc)
	22: (25,case)
	23: (25,amod)
	24: (25,compound)
	25: (2,nmod_in)
	26: (2,punct)

List((It,0), (can,1), (detect,2), (properties,3), (in,4), (Java,5), (,,6), (C,7), (,,8), (C++,9), (,,10), (Perl,11), (,,12), (and,13), (IOA,14), (programs,15), (,,16), (in,17), (spreadsheet,18), (files,19), (,,20), (and,21), (in,22), (other,23), (data,24), (sources,25), (.,26))
List((B-NP,0), (B-VP,1), (I-VP,2), (B-NP,3), (B-PP,4), (B-NP,5), (I-NP,6), (I-NP,7), (I-NP,8), (I-NP,9), (I-NP,10), (I-NP,11), (I-NP,12), (I-NP,13), (I-NP,14), (I-NP,15), (O,16), (B-PP,17), (B-NP,18), (I-NP,19), (O,20), (O,21), (B-PP,22), (B-NP,23), (I-NP,24), (I-NP,25), (O,26))

```

**Comment by MihaiSurdeanu** [comment]
Yeah, this sounds like a bug. I'll take a look.

**Comment by MihaiSurdeanu** [comment]
Fixed by PR #356 (this was a CoreNLP bug).
@marcovzla: The fix is in master now, but 8.0.0 has not been released yet.

**Comment by marcovzla** [comment]
Thanks!
